src/Microsoft.Azure.WebJobs.Extensions.Kafka/Trigger/KafkaTriggerMetrics.cs

```diff
@@ -0,0 +1,25 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using Microsoft.Azure.WebJobs.Host.Scale;
+using System;
+
+namespace Microsoft.Azure.WebJobs.Extensions.Kafka
+{
+    public class KafkaTriggerMetrics : ScaleMetrics
+    {
+        public long TotalLag { get; set; }
+        public long PartitionCount { get; set; }
+
+        public KafkaTriggerMetrics()
+        {
+        }
+
+        public KafkaTriggerMetrics(long totalLag, int partitionCount)
+        {
+            TotalLag = totalLag;
+            PartitionCount = partitionCount;
+            Timestamp = DateTime.UtcNow;
```

Comment:
Fixing my mistake here :). We can remove this line, as the base class does that.
https://github.com/Azure/azure-webjobs-sdk/blob/1e8479d748e0b13e5d957ad30447710ba0e3d7a3/src/Microsoft.Azure.WebJobs.Host/Scale/ScaleMetrics.cs#L15